docs/basics/publish-an-extension.md

```diff
@@ -18,13 +18,20 @@ Open your terminal, navigate to your extension directory and run `npm run build`
 
 To share your extension with others, you need to open a pull request in our [repository](https://github.com/raycast/extensions). For this, [fork our repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo), add your extension to your fork, push your changes and open a pull request [via the GitHub web interface](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) into our `main` branch.
 
-The extensions repository is big and to keep your git operations fast locally, you can do a sparse checkout with the following steps. This is especially useful when iterating on an existing extension.
+#### Alternative way
+The extensions repository is big and to avoid pulling all its content, you can do a sparse checkout with the following steps. This is also useful when iterating on an existing extension.
```

Comment:
> This is also useful when iterating on an existing extension.

This last part now make less sense, because the following steps are only for new extensions.
Maybe there is another place in the docs about iterating on existing extension, that we can add a slightly modified version of this to download only the extension you want to iterate on.